specification/dimensions/billingaccountname.md

```diff
@@ -0,0 +1,34 @@
+# Billing Account Name
+
+A billing account is a container for resources and/or services that are billed together in an invoice. Common scenarios for using billing accounts include grouping based on organizational constructs and hierarchy-based cost allocation approach.
+
+A billing account name is a display name assigned to a billing account. It is commonly used for cost reporting, invoice reconciliation, and cost allocation scenarios.
+
+The BillingAccountName column MUST be present in the billing data. This column MUST be of type String. The BillingAccountName MAY be a nullable column as some providers do not provide the ability to set a display name and in that case it MUST NOT duplicate the same value as the BillingAccountId. The BillingAccountName MUST NOT be null if a display name can be assigned to a billing account . BillingAccountName MUST be unique when a customer has more than one billing account.
```

Comment:
Why do we want to allow nulls? If someone groups by this, then it'll be strange to see that there's a charge that doesn't have a billing account name. Shouldn't we require it, even if the provider falls back to the ID? An ID is better than a null.

Reply:
I would agree with the not allowing nulls for Billing Account Name, especially where this is a more macro name vs a resource where there could be thousands of individual resources. We also are not allowing nulls on the sub account name so I think it makes sense not to allow nulls here either.

Reply:
How about 
```
"The BillingAccountName column MUST be present in the billing data. This column MUST be of type String. The BillingAccountName MUST NOT be null if a display name can be assigned to a billing account. BillingAccountName MUST be unique when a customer has more than one billing account." 
```
?